packages/ui/src/components/molecules/Topbar.tsx

```diff
@@ -123,13 +111,18 @@ export const Topbar: FC<{
         align="center"
         gap={16}
       >
-        <Image
-          src={isVisible ? visibilityIcon : visibilityHideIcon}
-          alt="visiblity"
-        />
+        {isVisible ? (
+          <Visibility fill={theme.palette.muted.main} />
+        ) : (
+          <VisibilityHide fill={theme.palette.muted.main} />
+        )}
       </Flex>
       <Flex px={2} py="3" align="center" gap={16}>
-        <Image src={isLock ? lock : lockOpen} alt="lock" />
+        {isLock ? (
+          <Lock fill={theme.palette.muted.main} />
+        ) : (
+          <LockOpen fill={theme.palette.muted.main} />
+        )}
```

Comment:
Do we need to provide the `fill` on every single icon? Doesn't it have a default fill?

Reply:
i can add a default fill, what do you suggest? im thinking white or muted

Reply:
on second thought that would be bad for components that only use a stroke color